CHANGELOG.md

```diff
@@ -1,6 +1,50 @@
 # Changelog
 
 
+## [1.52](https://github.com/brave/brave-ios/releases/tag/v1.52)
+
+### Web3
+
+- Added support for transactions from all networks on activity tab. ([#7233](https://github.com/brave/brave-ios/issues/7233))
```

Comment:
should be `Added support to show transactions from all networks in activity tab`